data/json/furniture.json

```diff
@@ -2081,6 +2146,7 @@
         "type" : "furniture",
         "id" : "f_vending_reinforced",
         "name": "reinforced vending machine",
+        "description": "You can't crack it open. It's reinforced.",
```

Comment:
I'm pretty sure they can be cracked open but it's damn hard. Maybe, `Good luck cracking this open.`